### DISCUSSION TITLE
如何向别人证明自己不是精神病

### DISCUSSION THREAD

**Discussion by giscus** [edony-ink/comments#29] [General]
# 如何向别人证明自己不是精神病

向别人证明一件事情最好的办法就是不要去证明，把自己还给自己，把别人还给别人。顺其自然，顺势而为。每个人只是宇宙的一粒尘埃，没有自己认为得那么重要。学会归位让生活步入正轨，按照自己的意愿和节奏生活才是正道，因为这个世界上最难的事情就是把自己的想法「放进」别人的脑袋。

https://www.edony.ink/how-to-prove-myself/

**Comment by Emotion04** [comment]
好文帮顶
